.github/workflows/pr-release.yaml

```diff
@@ -8,7 +8,7 @@ jobs:
     name: "Release PR to npm"
     runs-on: ubuntu-latest
     # comment out if:false to enable release PR to npm
-    if: false
+    # if: false
```

Comment:
I guess it slipped through.

Reply:
My tab was probably outdated. https://github.com/FuelLabs/fuels-ts/pull/1699/commits/3344c7d7a769177dc3a65f702f9ac0cb0719bdec